.golangci.yml

```diff
@@ -63,6 +63,8 @@ linters-settings:
     exclude: ./.errcheck-exclude.txt
   errorlint:
     errorf: false # don't check whether fmt.Errorf uses %w to wrap errors
+  goconst:
+    ignore-tests: true
```

Comment:
i think this is fine? it just turns of `goconst` for tests? i've had the linter find test issues before like failing to check an err return

Reply:
Yes, this only disables `goconst` in test files ([docs](https://golangci-lint.run/usage/linters/#goconst)). Without this setting `goconst` finds a lot of test parameters  that appear often in tests like "resource." or "span."